packages/fetchai/skills/tac_control_contract/game.py

```diff
@@ -843,23 +844,18 @@ def transactions(self) -> Transactions:
         """Get the transactions."""
         return self._transactions
 
-    def create(self, contract: Contract):
+    def create(self, configuration: Configuration):
```

Comment:
configuration is recoverable from the context isn't it, so no need to pass in argument?